reference/v8.json

```diff
@@ -394,7 +394,7 @@
       "type": "boolean",
       "function": "piecewise-constant",
       "default": false,
-      "doc": "If true, the icon will be visible even if it collides with other icons and text.",
+      "doc": "If true, the icon will be visible even if it collides with other previously drawn symbols.",
```

Comment:
Since UI-wise the layer ordering is flipped, I'm trying to avoid descriptions that reference layers as being "later/earlier" or "above/below" in the style. Maybe we should add a description in the general Layers section on how symbol layer priority works?  (On the UI side, we can explain it in guides).